CODEOWNERS

```diff
@@ -1,2 +1,4 @@
 # Default code owner for the repository
```

Comment:
In light of #59, might as well drop the CODEOWNERS commit and let the PR's rebase take care of this.

Reply:
Fixed Patch V3.